python/plugins/processing/algs/help/qgis.yaml

```diff
@@ -509,6 +509,13 @@ qgis:sumlinelengths: >
 qgis:texttofloat: >
   This algorithm modifies the type of a given attribute in a vector layer, converting a text attribute containing numeric strings into a numeric attribute.
 
+qgis:tilesxyz: >
+  This algorithm generates raster XYZ tiles of map canvas content.
+
+  Tile images can be saved as individual images in directory structure, or as single file in MBTiles format.
+
+  Tile size is fixed to 256x256.
```

Comment:
Some information about the output projection would be nice IMHO. Does it always output 3857? I guess yes, but it's good to mention.

BTW, the algorithm is missing tags. I was looking 'mbtiles' in the processing toolbox without finding it for example.

Nice addition, thanks.